extraMbedtls/generate-pkcs7.c

```diff
@@ -816,7 +817,7 @@ int to_pkcs7_already_signed_data(unsigned char **pkcs7, size_t *pkcs7Size, const
 		if (sigs[i]) free(sigs[i]);
 	}
 	if (sigs) free (sigs);
-
+	if (sig_sizes) free(sig_sizes);
```

Comment:
Are these fixes to a bug/leak introduced in the first commit of this PR? If so you can just squash this commit into that one, no need to have a separate commit for it.

If this is fixing an unrelated bug, ignore this.